akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/javadsl/ShardedDaemonProcess.scala

```diff
@@ -43,7 +47,7 @@ abstract class ShardedDaemonProcess {
       messageClass: Class[T],
       name: String,
       numberOfInstances: Int,
-      behaviorFactory: IntFunction[Behavior[T]]): Unit
+      behaviorFactory: IntFunction[Behavior[T]]): ActorRef[ShardedDaemonProcessCommand]
```

Comment:
Is this allowed binary compatible signature change? Otherwise we can add it only to the new `initWithContext`.

Reply:
Unsure, so backed on changing it in [d608bee](https://github.com/akka/akka/pull/31844/commits/d608bee914c17f59d688111263e58b77c0a601b9)